### PR TITLE
Add Manuel Cardenas' Bsc work

### PR DESCRIPTION
**Pull request by manoletre**

### CODE REVIEW

File: archive/2022/README.md [hunk 1]
```diff
@@ -19,3 +19,8 @@
...
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ---- | --------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Manuel Cardenas        | Multiple-Pose Augmented Learning System                                                                                       | BA   | José Vega                                                 | [PDF](winter/docs/bsc_cardenas_multiple_pose_augmented_learning_system.pdf)                                                                                     | [PDF](winter/talks/bsc_cardenas_multiple_pose_augmented_learning_system.pptx)                                                                                    |
```
Comment (mmisono) [inline archive/2022/README.md:24]: Is it possible to upload pdf instead of pptx?

Comment (mmisono) [reply archive/2022/README.md:24]: @manoletre ping?

Comment (manoletre) [reply archive/2022/README.md:24]: sorry, somehow missed your last messages. In the final presentation I created some videos, which displayed what I developed. I can replace them by pictures and create a pdf, but some information would go missing. As you prefer.

**Comment by manoletre** [comment]
Just changed pptx to pdf file and updated link of readme file

**Comment by mmisono** [comment]
@manoletre 
Thank you for the updates. However, unfortunately, since Git records everything, actually this PR contains pptx file in the commit b6073dd8 and thus the size of this PR is big. So can you remove the commit? (You need to run `git rm` and then force commit to github repository, or you can recreate a branch and create a new PR)

**Comment by manoletre** [comment]
created a new PR on #21 to avoid storing the big files

**Comment by manoletre** [comment]
Added work without history of videos on #21

**Pull request closed without merging**